src/main/core/manager.rs

```diff
@@ -533,7 +540,7 @@ impl<'a> Manager<'a> {
         let c_host = {
             let params = c::HostParameters {
                 // the manager sets this ID
-                id: unsafe { c::g_quark_from_string(hostname.as_ptr()) },
+                id: host_id,
```

Comment:
I think this also allows us to remove `g_quark_from_string` from main/build.rs.

Reply:
Done